lego/apps/meetings/search_indexes.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.postgres.search import SearchQuery, SearchVector
+
+from lego.apps.meetings.models import Meeting
+from lego.apps.meetings.serializers import MeetingSearchSerializer
+from lego.apps.search import register
+from lego.apps.search.index import SearchIndex
+
+
+class MeetingModelIndex(SearchIndex):
+
+    queryset = Meeting.objects.all()
+    serializer_class = MeetingSearchSerializer
+    result_fields = (
+        "title",
+        "description",
+        "report",
+        "start_time",
+    )
+    autocomplete_result_fields = ("title", "start_time")
+
+    autocomplete_fields = ("title", "description", "report")
```

Comment:
Remove `"report"` here. It doesn't make sense to autocomplete based on the content of the meeting report.

Reply:
The meeting report is the actual content of the meeting, while the description is the short message in calendars. I think it makes sense to be able to search for something from the report. (aka. the meeting notes). At least in our case, it is not very useful to only search for the meeting title. What do you think?

Reply:
Yes, that's why it makes sense to have it in `search_fields`, which is exactly what this field is for (as you have done). But not in `autocomplete_fields`, as that would just be confusing and not make sense.

Reply:
Hm, that is weird. In my testing, I have to include "report" in the autocomplete_fields as well to make it actually show up in the results. 🤔

Reply:
So, the reason for this is kinda weird/stupid. In lego-webapp, when you search in the top bar, and wait for results to show up as you type, we use the `/autocomplete` endpoint, and the full search is only done once you press Enter.

This is kinda strange and confusing, and we have discussed before what the best behavior is here, but haven't changed it, or made it more clear that the search-as-you-type results aren't full search. The behavior I describe we want for meetings, is also how it works for events, pages, articles etc. If you search for page content in lego-webapp now, you'll notice you won't get any results, but when you press enter, you will see the expected results.

Reply:
It does say "Trykk enter for fullstendig søk at the bottom of the results" but it's not clear enough what the behavior is I guess :upside_down_face:.

Reply:
Ah yes, I see. Then it makes sense, I agree.